### DISCUSSION TITLE
Option to "Send Item Data" (all the fields) in a webhook or flow?

### DISCUSSION THREAD

**Discussion by luciandavidescu** [directus/directus#17030] [Archive]
Right now, with Webhooks' "Send Event Data" (or with Flows), a limited subset of an item's data is sent over. How about an option to send an item's full content, with all the fields and everything?

It would be great for things like outputting webpages on the fly, sending e-mail alerts or push notifications.

(surely it can be done externally to some extent, but in time that gets quite unreliable)

**Comment by scottybo** [comment]
@luciandavidescu did you find a solution for this?

**Reply by luciandavidescu** [reply]
Not quite, but the same purpose can be acheved by making another trip to the DB with a 'read data' block, passing the id of the item.

**Comment by br41nslug** [comment]
![image](https://user-images.githubusercontent.com/9389634/216010408-3997a9fb-e0ab-4987-b329-3a92ca99cd38.png)

> but the same purpose can be acheved by making another trip to the DB with a 'read data' block, passing the id of the item.

whether done in the core or like above a new database query is always needed to get the "full" object as default values are handled at the database level.
